hudi-client/hudi-client-common/src/test/java/org/apache/hudi/table/action/rollback/TestMarkerBasedRollbackStrategy.java

```diff
@@ -141,7 +141,7 @@ private Map<String, Long> createLogFilesAndMarkersToRollback(String partition,
       } catch (IOException e) {
         throw new RuntimeException(e);
       }
-    }).collect(Collectors.toMap(Function.identity(), e -> 1L));
```

Comment:
why flipping the size here? should we expect the size to be non 0 somewhere down the line and hence we added 1 as length here.

Reply:
https://github.com/apache/hudi/blob/3de3769ea78e0ca2a5dd1c682c26fb4d67c77e59/hudi-client/hudi-client-common/src/test/java/org/apache/hudi/table/action/rollback/TestMarkerBasedRollbackStrategy.java#L131-L132 creates a log file with size 0. So using the same while returning.